### DISCUSSION TITLE
What would you like to see in the application

### DISCUSSION THREAD

**Discussion by RevanderKh** [stoatchat/discussions#810] [Feature Suggestions]
I would really like to see the auditor's log on the server, server templates, user links, soundbars, news channels (with the ability to automatically send to channels where a subscription was made), a welcome screen. That seems to be all for now

**Comment by RevanderKh** [comment]
And I forgot, I would also like to see the server folders to simplify the orientation of the application, and it was possible to sort all channels by folders.

**Comment by RevanderKh** [comment]
I would also like the system of mentioning roles to be the same as users

**Comment by AsakiQ** [comment]
Registration of permanent links. This needs to be done as quickly as possible in order to pull the audience from Discord to Revolt.

**Comment by savesove** [comment]
Since many servers are now switching to this platform, adding an individual custom link would make it easier to join popular servers

**Comment by Alex11381** [comment]
Sharing screen

**Comment by RevanderKh** [comment]
Еще хотелось бы видеть более удобное форматирование текста, а также гораздо большее его разнообразие чем в Discord

**Comment by Monechk** [comment]
Я немного теряюсь, извините. Здравствуйте! Добавьте, пожалуйста, возможность закрепить важные сообщения в беседах, это очень важно и необходимо, чтобы не потерять и ещё создать в каналах "#" подветви, которые ведут обсуждения внутри определенных сообщений. Это тоже очень нужная функция. Более пока ничего не волнует

**Comment by RevanderKh** [comment]
I don't remember if I wrote about it, but it would be cool to add the ability to mention roles, and make the whole system of mentions easier, otherwise the same output of the user's ID instead of his usual nickname is not convenient

**Comment by TarLink48** [comment]
Noise reduction like Krisp or something similar on AI to not hear the pets of the interlocutor or like I eat chips )

**Comment by SvetlanaGuseva** [comment]
по возможности добавить возможность загружать картинки большего размера по ссылке или как то иначе.. + возможность создавать ботов для генерации артов как  вдискорде! только на своих РФ серверах

**Comment by Fressky** [comment]
Add screen sharing, as well as animation of avatars during a conversation (to understand that there is sound)
